### ISSUE TITLE
mocha-no-side-effect-code: bug with it.skip triggering false positive

### ISSUE THREAD

**Issue by ethanresnick** [Microsoft/tslint-microsoft-contrib#441]
Test case:

```ts
describe("My suite", () => {
    it.skip("my test so skip");
});
```

*Expected behavior*: no error; the `it.skip` call is safe.

*Actual behavior*: mocha-no-side-effect-code reports an error on the `it.skip` line.

**Comment by ethanresnick** [comment]
Sorry, it looks like this is a duplicate of #389, which has been fixed. Closing.
